src/lib/prompts.ts

```diff
@@ -13,6 +13,8 @@ interface Spec {
   title: string;
 }
 
+export type OpenapiPromptOptions = 'create' | 'update';
```

Comment:
super nit but can you rename this?

```suggestion
export type OpenAPIPromptOptions = 'create' | 'update';
```

Reply:
i always go back and forth on these things lol